image-processing/UI.py

```diff
@@ -1,484 +1,202 @@
-                        print("Please ensure at least one file is selected for analysis before proceeding. ")
-                    
-                    elif filename_input.lower() == 'stop':
-                        break
-
-
-                    elif filename_input.lower() == 'clear':
-                        scans_dir = os.listdir(scans_folder)
-                        for npyscan in scans_dir:
-                            if npyscan.endswith(".npy"):
-                                os.remove(os.path.join(scans_folder, npyscan))
-
-                        print("Saved files cleared.")
-
-                    else:
-                        filename_npy = filename_input + '.npy'
-                        file_path = scans_folder / filename_npy
-
-                        if not file_path.exists():
-                            print("Sorry, this file does not exist. Please re-enter including any spaces, and the file extension, e.g. .jpeg")
-                            continue
-
-                        else:
-                            while True:
-                                grouping = input("is the file '{}' a scan of a low, or high areal weight sample? ".format(filename_input)).lower()
-                                if not (grouping == 'high' or grouping =='low'):
-                                    print('response not recognised, please respond with either high or low')
-                                    continue
-                                break
-                            file = np.load(file_path, allow_pickle = True)
-                            paths_saved.append(file_path)
-                            scans_saved.append([file,grouping])
-
-                            print("File added successfully.")
-                        no_file = True
-                            # print(paths_saved)
-
-
-                print("Please respond 'yes' or 'no' to the following questions:")
-
-                while True:
-                    view_plot = input("Would you like to view the output plot from the analysis software? ").lower()
-
-                    if not view_plot in ('yes', 'no'):
-                        yesno_error()
-                        continue
-
-                    break
-
-                while True:
-                    set_parameters = input("Would you like to use preset values or set your own parameters? ('yes' for presets, 'no' for custom parameters) ").lower()
-
-                    if not set_parameters in ('yes', 'no'):
-                        yesno_error()
-                        continue
-
-                    break
-
-
-                for scan in range(len(scans_saved)):
-
-                    if set_parameters == 'yes':
+    def input_sample(self):
+        sample_type = input("Type of Sample: ")
+        
+        if sample_type in [str(n) for n in range(len(self.material_types) + 1)]:
+            print("Please ensure samples are scanned at 75dpi, or the next closest possible resolution.\n")
```

Comment:
I think any resolution deviation my lead to incorrect values...

Reply:
just realised i can actually reply to these... sorted this issue!